src/dash-table/derived/edges/type.ts

```diff
@@ -26,10 +26,19 @@ export const BORDER_PROPERTIES: BorderProp[] = [
     'borderTop'
 ];
 
+const cssProperties = Array.from(py2jsCssProperties.values());
+
 export const BORDER_PROPERTIES_AND_FRAGMENTS: string[] = R.uniq(
     R.filter(
         p => p.indexOf('border') === 0,
-        Array.from(py2jsCssProperties.values())
+        cssProperties
+    )
+);
+
+export const HEIGHT_AND_WIDTH_PROPERTIES: string[] = R.uniq(
```

Comment:
heh could get rid of these `R.uniq` calls too if `py2jsCssProperties` was rewritten 🌴 and exposed the unique list too 😉